frontend/src/routes/Infrastructure/Clusters/ClusterSets/ClusterSetDetails/ClusterSetInstallSubmariner/InstallSubmarinerForm.tsx

```diff
@@ -378,23 +378,12 @@ export function InstallSubmarinerForm(props: { availableClusters: Cluster[] }) {
         }
 
         if (anyUnsupported) {
-            resources.push({
-                apiVersion: ManagedClusterAddOnApiVersion,
-                kind: ManagedClusterAddOnKind,
-                metadata: {
-                    name: 'submariner',
-                    namespace: 'roks-cluster',
-                },
-                spec: {
-                    installNamespace: 'submariner-operator',
-                },
-            })
             const broker: Broker = {
                 apiVersion: BrokerApiVersion,
                 kind: BrokerKind,
                 metadata: {
                     name: 'submariner-broker',
```

Comment:
Please use the constant `defaultBrokerName`.

Reply:
thanks @dberg1 !